installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc

```diff
@@ -31,7 +31,9 @@ include::modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc[leve
 
 * xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backing-up-etcd-data_backup-etcd[Backing up etcd]
 
-* xref:../../post_installation_configuration/bare-metal-configuration.adoc#post-install-bare-metal-configuration[Bare metal configuration] 
+* xref:../../post_installation_configuration/bare-metal-configuration.adoc#post-install-bare-metal-configuration[Bare metal configuration]
+
+* xref:../installing_bare_metal_ipi/ipi-install-installation-workflow.html#bmc-addressing_ipi-install-installation-workflow[BMC addressing]
```

Comment:
The `.html` should be an `.adoc` in the `xref`. It will work on the docs.openshift.com build, but causes problems on the Customer Portal build.

Reply:
Thanks Alex, updated the PR